setup.py

```diff
@@ -22,7 +22,7 @@
 
 
 _deps = [
-    "transformers>=4.27.4",
+    "transformers>=4.27.4,<4.35.0",
```

Comment:
Transformers 4.35.0 switches to loading from safetensors for Flax by default. We can't currently load from safetensors with `_do_init=False`, hence we pin the requirement temporarily